docs/publishers/core-datasets.md

```diff
@@ -0,0 +1,135 @@
+# Core Datasets
+
+**Core Datasets** - important, commonly-used datasets in high quality, easy-to-use & open form.
```

Comment:
I did some research using google Ngram Viewer, so "commonly used" would be better to use.
https://books.google.com/ngrams/graph?content=commonly+used%2Ccommonly-used&year_start=1900&year_end=2000&corpus=18&smoothing=3&share=&direct_url=t1%3B%2Ccommonly%20used%3B%2Cc0%3B.t1%3B%2Ccommonly%20-%20used%3B%2Cc0.